### DISCUSSION TITLE
✅ Add the "Shantell Sans" Google Font - [ED-14579]

### DISCUSSION THREAD

**Discussion by Countmooshroom** [elementor/elementor#25095] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I'd really like to use the font Shantell Sans soon, which is a Google font, but it's not yet built-in to Elementor.

### Describe the solution you'd like.

I'd like Elementor to include the Shantell Sans font as a supported font that I can use automatically.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @Countmooshroom!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.22.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
